rsearch/bin/main.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	search "github.com/romana/contrib/rsearch"
+	"log"
+	//	"net/http"
+	"encoding/json"
+
+	//	"io"
+)
+
+func main() {
+	var cfgFile = flag.String("c", "", "Kubernetes reverse search config file")
+	var server = flag.Bool("s", false, "Start a server")
+	var host = flag.String("h", "", "Protocol://host for client to connect to")
```

Comment:
A partial URL is neither a host or a URL.
You should pick one -- either a full URL (including port), or a hostname and port that you'll compose into a URL.